samples/kubernetes-event-source/README.md

```diff
@@ -87,21 +86,36 @@ kubectl delete pods busybox
 
 ### Verify
 
-We will verify that the kubernetes events were sent into the Knative eventing
-system by looking at our message dumper function logsIf you deployed the
+We will verify that the Kubernetes events were sent into the Knative eventing
+system by looking at our message dumper function logs. If you deployed the
 [Subscriber](#subscriber), then continue using this section. If not, then you
```

Comment:
Personally I would remove the instructions for people who haven't installed the `Subscriber` as it's not that helpful.

Reply:
Good idea, done.